### ISSUE TITLE
Issue tracker shows wrong results with e.g. openSUSE/umoci#142 in the changelog

### ISSUE THREAD

**Issue by MaximilianMeister** [openSUSE/open-build-service#3725]
Issue/Feature description
====================

Issue tracker shows wrong results (to openoffice), probably when a mentioned issue has an `i` before the `#`.

mentioned issue: `openSUSE/umoci#142`
link in the obs UI: [i#142](http://openoffice.org/bugzilla/show_bug.cgi?id=142)

Expected result
============

either point to the github issue, or filter the result, as it's not a bugzilla issue

How to Reproduce
===============
1. When submitting a changelog with e.g. `openSUSE/umoci#142`
2. the issue tracker wrongly displays the link

Further information
===============
* https://build.suse.de/request/show/139397

**Comment by ChrisBr** [comment]
Yep, see here 
https://github.com/openSUSE/open-build-service/blob/master/src/api/db/seeds.rb#L256

Hm, we would need to have a generic GitHub issue tracker. Something like ``github#openSUSE/umoci#142``. However, this is a little bit ugly as we always also need to add the project path to the issue. 

@hennevogel @adrianschroeter any good idea?

**Comment by hennevogel** [comment]
Any string could be in the project path right? You can only make this less likely by using longer string...

**Comment by ChrisBr** [comment]
Yup, any string can be the path! In the past we created for specific projects a dedicated issue tracker e.g.  obs#42 will link to openSUSE/open-build-service#42. However, I like the idea to have an generic github issue tracker.

**Comment by adrianschroeter** [comment]
The question is for me if ne1 will still understand and use such a complex syntax. In the end it is supposed to get written into the changes and _patchinfo files manually. And a syntax error can happen easily in free text fields.

Maybe we should consider just to match on the full URL to a github issue ...

**Comment by ChrisBr** [comment]
And maybe we should add to the issue trackers regex ``^`` and ``$``.

**Comment by eduardoj** [comment]
This should have been resolved by #6875. Closing. Please reopen if the issue applies.
